modules/ting_search/ting_search.module

```diff
@@ -811,10 +824,21 @@ function search_extend_content() {
 function ting_search_sort_form($form_state) {
   $form = array();
 
+  // Try to determine current sort.
+  if (NULL !== ting_search_current_results() && !empty(ting_search_current_results()->getSearchRequest()->getSorts())) {
```

Comment:
*Perhaps* we can leave out the `NULL` check here as well?

Reply:
Had the thought but could not really do it in good conscience - so I left it in there.

Reply:
:+1: